web/tendrl/admin_user_permissions.rst

```diff
@@ -0,0 +1,131 @@
+Multiuser - Positive login
+**************************
+
+:authors:
+          - ltrilety@redhat.com
+
+Description
+===========
+
+**UNDER CONSTRUCTION - NEED UPDATE**
+
+Login as user with 'Admin' role.
+Check that all features what should be available really are and vice versa.
+
+Setup
+=====
+
+
+Test Steps
+==========
+
+.. test_action::
+    :step:
+        Log as Admin user
+    :result:
+        User should be logged on. There should be availalble 'Clusters' list,
+        'Hosts' list, 'Alerts' list, 'Notifications' and 'Tasks' list.
+        Users list/menu should be available.
+
+.. test_action::
+    :step:
+        Open Hosts list.
+    :result:
+        Filter and order fields should be available and working.
+
+.. test_action::
+    :step:
+        Open Alerts list
+    :result:
+        Filter and order fields should be available and working.
+
+.. test_action::
+    :step:
+        Open Notifications list
+    :result:
+        Filter and order fields should be available and working.
+
+.. test_action::
+    :step:
+        Open Tasks list
+    :result:
+        Filter and order fields should be available and working.
+        It should be possible to open task details.
+
+.. test_action::
+    :step:
+        Open Clusters list page
+    :result:
+        Filter and order fields should be available and working.
+        There should be functional link to Grafana for imported clusters.
+        It should be possible to import, manage, unmanage and forget cluster.
```

Comment:
Why do you think that Tendrl will provide unmanage or forget cluster functionality? As far as I know this feature would not be directly available in the ui, at least for the upcoming upstream release we are targeting in these test cases. I would suggest to either link to the specification/issue/ux desing for this feature or to remove this from the test case. Moreover it would be nice to link to the definition of what manage, unmanage and forget actually mean in this context (without that, this explanation is not very useful wrt to expected functionality).